tests/core/pyspec/eth2spec/test/electra/block_processing/test_process_attestation.py

```diff
@@ -79,3 +83,30 @@ def test_invalid_nonset_committe_bits(spec, state):
     attestation.committee_bits[committee_index] = 0
 
     yield from run_attestation_processing(spec, state, attestation, valid=False)
+
+
+@with_electra_and_later
+@spec_state_test
+@with_presets([MINIMAL], "need multiple committees per slot")
+@always_bls
+def test_multiple_committees(spec, state):
+    attestation_data = build_attestation_data(spec, state, slot=state.slot, index=0)
+    attestation = spec.Attestation(data=attestation_data)
+
+    # fill the attestation with two committees and finally sign it
+    fill_aggregate_attestation(spec, state, attestation, signed=False, committee_index=0)
+    fill_aggregate_attestation(spec, state, attestation, signed=True, committee_index=1)
```

Comment:
Why is the first unsigned & the second signed?

Reply:
The second one signs the attestation with two committees — the desired scenario, there is no need to sign with just one so I decided to skip it for the performance sake (not sure if the gain is big though as signing is cheap). Might want to sign for consistency?

Reply:
Oh I see... yes, signing is cheap. I'd prefer we sign both times, but a comment would be okay too.